### ISSUE TITLE
Download table in different formats

### ISSUE THREAD

**Issue by DanielAndreasen** [DanielAndreasen/SWEETer-Cat#35]
Although it is nice with the tools online, it is often more convinient to have them locally.
Let's make a download button to download the table in different formats: `csv`, `tsv`, `fits`?, others?

**Comment by jason-neal** [comment]
Would it be any use as a `fits` file? I only use them for spectra....

What about as an `hdf5` format. 

> It is designed for flexible and efficient I/O and for high volume and complex data.

Maybe  `hdf5` format can also help with the home page loading?

**Comment by DanielAndreasen** [comment]
I know about `hdf5` format, but I have never used it. I really don't have any strong opinions about this. If I can just read it with `pandas` I will be satisfied.

As for the rendering problem (#19), I don't think another format will change anything.

**Comment by jason-neal** [comment]
Pandas has both  `pd.read_hdf ` and  `pd.to_hdf`

**Comment by DanielAndreasen** [comment]
I would also like to just have one table, and dynamically generate other formats with `pd.to_<some_format>`. Sounds like a better solution since the tables are not that big (2500+ rows and less than 30 columns).

**Comment by jason-neal** [comment]
I was trying something like this becuase I was wanting to profile `readSC()` when using the hdf format but I was having issues with hdf, not finding` pytables` although I have installed it.... I give up for now.
```
def table_convert(df, fmt="csv", name="data/test_data" ):
    """Convert table to different formats."""
    if fmt is "hdf":
        df.to_hdf(name + "." + fmt, key="sweetcat", mode="w")
    elif fmt is "csv":
        df.to_csv(name + "." + fmt, sep=",")
    else:
        NotImplementedError("Conversion format to {} not available.".format(fmt))

```

**Comment by DanielAndreasen** [comment]
Yes, this is something along the lines I want. I'll have a look later with this. Thanks for sharing. Feel free to make a commit to the download branch even though it doesn't work yet.

**Comment by DanielAndreasen** [comment]
Solved in PR #36
